features/division.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Division
+  As a mathematician
+  I want my calculator to divide numbers
+  So I can know their result
+
+ Scenario: should return an integer
+  	When I divide 2 numbers

```

Comment:
This spacing is incorrect. Ruby is a 2 space (no tabs) language by convention. No points are being taken off. Just be on the lookout for these.